src/ProductConstructionService/ProductConstructionService.Api/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -318,7 +323,11 @@ await _context.BuildDependencies.AddRangeAsync(
 
         // Compute the dependency incoherencies of the build.
         // Since this might be an expensive operation we do it asynchronously.
-        // TODO (https://github.com/dotnet/arcade-services/issues/3814): Queue.Post<BuildCoherencyInfoWorkItem>(JToken.FromObject(buildModel.Id));
+        await _workItemProducerFactory.Create<
+            BuildCoherencyInfoWorkItem>().ProduceWorkItemAsync(new()
+        {
+            BuildId = buildModel.Id
+        });
```

Comment:
The indentation here is broken up pretty bad 😅

Reply:
ops ahah, fixed now